openreview/tools.py

```diff
@@ -17,6 +17,14 @@ def get_profile(client, value):
             return e
     return profile
 
+def get_profile_id(client, value):
```

Comment:
I don't think that we need a function for this, get_profile_id doesn't return always an id. So the name is confusing. 

I would replace the function call with these 2 lines:

```
profile = tools.get_profile(client, user)
user = profile.id if profile else user
```